install/traf_checkset_env.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
+#
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+#
+# make sure the environment can build the code of trafodion or not.
+# must configure the yum repo right before execute this script.
+# run this script with normal user, while must has sudo permission.
+
+#default path
```

Comment:
This command needs a help text that should mention the following:

- Who should use this script (people who want to set up a single node
  development system on a machine where they have sudo privileges)
- What does the script do (e.g. yum install, build/setup tools).
- A link to the Contributor guide wiki page that describes these steps
  (https://cwiki.apache.org/confluence/display/TRAFODION/Create+Build+Environment)
- What environment variables must be set before calling this script
- What optional environment variables can be set
- What directories are created by the script, and how they can be customized

Once this PR is merged, we can also update the wiki page and mention this command.